### PR TITLE
개선

### PR DESCRIPTION
**Pull request by wonderlandpark**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,47 @@
 
 ## 더 이상 봇 인증을 받지 않습니다.
```
Comment (zero734kr) [inline README.md]: why didn't you remove it?

File: README.md [hunk 2]
```diff
@@ -2,4 +2,47 @@
...
+## 목적
+
+- 디스코드 인증보다 더 자세한 기준으로 신뢰성을 주기 위해서입니다.
+
+## 기준
+
+신뢰할 수 있는 봇임을 증명하려면 아래의 기준에 모두 해당해야합니다.
+
+- 5000개 이상의 서버
+- 유저가 지원받을 수 있는 방법이 존재
```
Comment (zero734kr) [inline README.md]:
```suggestion
- 유저가 공식적인 지원 통로(웹사이트, 티켓 채널 등)를 통해 문의할 수 있는 방법이 존재
```

File: README.md [hunk 3]
```diff
@@ -2,4 +2,47 @@
...
+- 수집하는 개인정보와 이용범위를 서비스에 명시
```
Comment (zero734kr) [inline README.md]:
```suggestion
- 수집하는 개인정보, 이용범위, 제 3자 위탁 업체를 유저가 접근 가능한 약관에 명시

File: README.md [hunk 4]
```diff
@@ -2,4 +2,47 @@
...
+- 평균 90퍼센트 이상의 업타임
```
Comment (zero734kr) [inline README.md]: I think 95% would be better

File: README.md [hunk 5]
```diff
@@ -2,4 +2,47 @@
...
+- 안전하고 신뢰할 수 있는 저장방식
```
Comment (zero734kr) [inline README.md]:
```suggestion
- 안전하고 신뢰할 수 있는 개인 정보 같은 데이터의 저장 방식
```

File: README.md [hunk 6]
```diff
@@ -2,4 +2,47 @@
...
+- 디스코드 인증된 봇
```
Comment (zero734kr) [inline README.md]:
```suggestion
- 디스코드에게 인증된 봇
```

File: README.md [hunk 7]
```diff
@@ -2,4 +2,47 @@
...
+혹은 아래 기준 중 하나 이상에 해당해야합니다.
+
+- 디스코드 인증과 파트너쉽이 체결된 봇.
+- 당사가 인정한 특수 목적의 봇.
+
+## 혜택
+
+신뢰된 봇은 다음 혜택을 가집니다.
+
+- 봇의 Vanity URL, 배너, 배경 커스터마이징
```
Comment (zero734kr) [inline README.md]:
```suggestion
- 봇의 전용 고유 URL, 배너와 배경 커스터마이징
```

File: README.md [hunk 8]
```diff
@@ -2,4 +2,47 @@
...
+- 당사에 직접적으로 연락할 수 있는 창구
+- 신규 기능 미리 공개
+
+## 책임
+
+- 인증된 봇이여도, 당사는 어떠한 보증과 책임을 일체 부인합니다.
```
Comment (zero734kr) [inline README.md]:
```suggestion
- 인증된 봇임에도 불구하고 봇 또는 봇의 개발자가 사회적 또는 도의적 물의를 일으킬 경우 인증된 봇 자격 박탈 외에는 어떠한 보증과 책임을 일체 부인합니다.
```

File: README.md [hunk 9]
```diff
@@ -2,4 +2,47 @@
...
+- 인증된 봇은 당사와 어떠한 관련이 있지 않음을 밝힙니다.
```
Comment (zero734kr) [inline README.md]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -2,4 +2,47 @@
...
+
+## 신청
+
+지정된 양식으로 맞춰 이슈를 생성해주세요.
+
+## 기타
+
+- 추후 기준이 변경되어도 가이드라인을 위반하지 않는 한 신뢰된 봇 자격이 박탈되지는 않습니다.
```
Comment (zero734kr) [inline README.md]:
```suggestion
- 당사는 가이드라인 및 이용자들에게 물의를 일으켜 봇에 대한 신뢰성에 심각한 타격을 줄 경우 신뢰된 봇 자격을 박탈할 권한을 갖습니다.
```

File: README.md [hunk 11]
```diff
@@ -2,4 +2,47 @@
...
+- 신뢰된 봇 인증 이후에 지속적으로 기준에 미달하는 경우 당사는 신뢰된 봇 자격을 박탈할 권한을 갖습니다.
+- 당사는 공개된 기준 이외에 내부 심사를 통해 승인 여부를 결정합니다.
```
Comment (zero734kr) [inline README.md]:
```suggestion
- 당사는 공개된 조건 이외에 봇에 대한 신뢰성을 보장할 수 있는지 대한 내부 심사를 통해 승인 여부를 결정합니다.
```

**Review by zero734kr — CHANGES_REQUESTED** [review]

**Review by kijk2869 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/submit.yml [hunk 12]
```diff
@@ -0,0 +1,105 @@
+name: 봇 인증 신청
+description: 봇 인증을 신청합니다.
```
Comment (zero734kr) [inline .github/ISSUE_TEMPLATE/submit.yml]:
```suggestion
description: 봇의 인증을 신청합니다.
```

File: .github/ISSUE_TEMPLATE/submit.yml [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+title: 봇 이름으로 변경해주세요
```
Comment (zero734kr) [inline .github/ISSUE_TEMPLATE/submit.yml]:
```suggestion
title: (봇 이름으로 변경해주세요)
```

File: .github/ISSUE_TEMPLATE/submit.yml [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+    attributes:
+      label: 수집하는 개인정보
+      description: 봇이 수집하는 개인정보는 무엇인가요?
+    validations:
+      required: true
+  - type: textarea
+    id: privacy-policy
+    attributes:
+      label: 개인정보에 관한 안내
+      description: 개인정보에 관한 안내는 서비스에 어떻게 안내되어 있나요? 또한 개인정보취급방침을 기재해주세요!
```
Comment (zero734kr) [inline .github/ISSUE_TEMPLATE/submit.yml]:
```suggestion
      description: 개인정보에 관한 안내는 서비스에 어떻게 안내되어 있나요? 또한 개인정보 취급 방침을 기재해주세요!
```

**Review by zero734kr — CHANGES_REQUESTED** [review]

**Review by zero734kr — APPROVED** [review]

**Pull request merged**